src/worker.cpp

```diff
@@ -486,11 +486,32 @@ void Worker::setDeviceTreeAndJson()
 
     if (fitConfigVal.find(devTreeFromJson) != std::string::npos)
     { // fitconfig is updated and correct JSON is set.
-
         if (isSystemVPDOnDBus())
         {
-            // TODO
-            //  Restore system VPD logic should initiate from here.
+            m_backupRestoreObjPtr = new BackupRestore(m_parsedJson);
```

Comment:
Why is this needed and where it is getting freed.

Reply:
only including BackupRestore API's implementation in this PR.